package.json

```diff
@@ -18,12 +18,10 @@
     "@angular/compiler": "15.1.0",
     "@angular/core": "15.1.0",
     "@angular/forms": "15.1.0",
-    "@angular/material": "15.1.0",
     "@angular/platform-browser": "15.1.0",
     "@angular/platform-browser-dynamic": "15.1.0",
     "@angular/router": "15.1.0",
     "date-fns": "2.29.3",
-    "moment": "2.29.4",
```

Comment:
SuperMinor: Don't need to work with dates anymore?

Reply:
No, this lib doesn't need anymore